### PR TITLE
Replaced all occurrences of userShouldBeSearched and its reducer

### PR DESCRIPTION
**Pull request by 0xNilesh**

### CODE REVIEW

**Review by corlard3y — APPROVED** [review]

**Review by corlard3y — CHANGES_REQUESTED** [review]
Kindly fix conflicts @0xNilesh

**Review by corlard3y — APPROVED** [review]

File: src/components/chat/w2wChat/newusers/NewUser.tsx [hunk 1]
```diff
@@ -32,8 +32,8 @@ function NewUser() {
     hasUserBeenSearched,
     setHasUserBeenSearched,
     setActiveTab,
-    userShouldBeSearched,
-    setUserShouldBeSearched,
+    // userShouldBeSearched,
```
Comment (mishramonalisha76) [inline src/components/chat/w2wChat/newusers/NewUser.tsx:35]: remove these unsused variables here

File: src/components/chat/w2wChat/searchBar/SearchBar.tsx [hunk 2]
```diff
@@ -50,25 +52,29 @@ const SearchBar = () => {
...
+
   const {
     setSearchedUser,
     searchedUser,
     hasUserBeenSearched,
     setHasUserBeenSearched,
     setActiveTab,
-    userShouldBeSearched,
-    setUserShouldBeSearched,
+    // userShouldBeSearched,
```
Comment (mishramonalisha76) [inline src/components/chat/w2wChat/searchBar/SearchBar.tsx:63]: remove these unused ones

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]

File: src/components/chat/w2wChat/newusers/NewUser.tsx (hunk 1 above)
Comment (0xNilesh) [reply src/components/chat/w2wChat/newusers/NewUser.tsx]: done

File: src/components/chat/w2wChat/searchBar/SearchBar.tsx (hunk 2 above)
Comment (0xNilesh) [reply src/components/chat/w2wChat/searchBar/SearchBar.tsx]: done

**Pull request merged**